java/client/src/main/java/glide/api/RedisClusterClient.java

```diff
@@ -446,4 +447,16 @@ public CompletableFuture<String> functionLoadReplace(
                 route,
                 this::handleStringResponse);
     }
+
+    @Override
+    public CompletableFuture<String> randomKey(@NonNull SingleNodeRoute route) {
```

Comment:
@cyip10 after further experimentation, we should change this to map to anything, since users should know what they're doing when they pick a route.  The ALL_PRIMARIES (default) is probably the best solution in 99.9% of the cases. 
```suggestion
    public CompletableFuture<String> randomKey(@NonNull Route route) {
```